core/store/migrate/migrations/0223_log_poller_filters_add_topics_logs_per_block.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
+
+ALTER TABLE evm.log_poller_filters
+    ADD COLUMN topic2 BYTEA CHECK (octet_length(topic2) = 32),
+    ADD COLUMN topic3 BYTEA CHECK (octet_length(topic3) = 32),
+    ADD COLUMN topic4 BYTEA CHECK (octet_length(topic4) = 32),
+    ADD COLUMN max_logs_kept NUMERIC(78,0),
```

Comment:
10^77 of logs? Does it make sense? Isn't regular bigint just enough? I can't imagine a case in which someone defines that he/she wants to keep 10*18 logs or sth :D